### PR TITLE
EVG-6087 add limit to perf result by name route

### PR DESCRIPTION
**Pull request by john-m-liu**

### CODE REVIEW

File: model/perf.go [hunk 1]
```diff
@@ -272,7 +273,11 @@ func (r *PerformanceResults) Find(options PerfFindOptions) error {
 	}
 
 	r.populated = false
-	err = session.DB(conf.DatabaseName).C(perfResultCollection).Find(search).All(&r.Results)
+	query := session.DB(conf.DatabaseName).C(perfResultCollection).Find(search).Sort("-" + perfCreatedAtKey)
```
Comment (tychoish) [inline model/perf.go]: is there an index for this?

File: rest/perf_routes.go [hunk 2]
```diff
@@ -8,6 +8,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/mongodb/grip"
+
```
Comment (tychoish) [inline rest/perf_routes.go]: blank line

Comment (john-m-liu) [reply rest/perf_routes.go]: fixed

File: model/perf.go (hunk 1 above)
Comment (john-m-liu) [reply model/perf.go]: There's a created_at index but since this query can be many shapes it may or may not be used

**Pull request merged**